src/interval.jl

```diff
@@ -197,7 +265,14 @@ function Base.:isless(a::AbstractInterval, b::AbstractInterval)
 end
 
 function less_than_disjoint(a::AbstractInterval, b::AbstractInterval)
-    return RightEndpoint(a) < LeftEndpoint(b)
+    re = RightEndpoint(a)
+    le = LeftEndpoint(b)
+
+    if isnothing(re) || isnothing(le)
+        return false
+    else
+        return re < le
+    end
```

Comment:
Logic shouldn't have to change here as endpoint comparisons should handle unbounded right?

Reply:
I'll have to double check. At some point I found that some of the tests were failing here without modifications, but I'll go back to it and make sure